src/app/scripts/cac/pages/cac-pages-home.js

```diff
@@ -125,12 +130,41 @@ CAC.Pages.Home = (function ($, BikeModeOptions, Templates, UserPreferences) {
 
     return Home;
 
-    function clickedViewAll(event) {
+    function clickedViewAllArticles(event) {
+        event.preventDefault();
+
+        // hide existing articles and show loading spinner
+        $(options.selectors.articlesContainer).addClass('hidden');
+        $(options.selectors.articlesSpinner).removeClass('hidden');
+
+        $.ajax({
+            type: 'GET',
+            cache: true,
+            url: articleUrl,
+            contentType: 'application/json'
+        }).then(function(data) {
+            if (data && data.length) {
+                var html = Templates.articles(data);
+                $(options.selectors.articlesContainer).html(html);
+
+                // hide 'view all' button and spinner, and show features again
+                $(options.selectors.viewAllArticles).addClass('hidden');
+                $(options.selectors.articlesSpinner).addClass('hidden');
+                $(options.selectors.articlesContainer).removeClass('hidden');
+            } else {
+                console.error('Could not load all articles');
```

Comment:
We've got several references to `console.error` in the code base.  I thought the idea was to avoid `console.log` for expected conditions?